{"completed": [], "queue": [], "results": [{"project": "drf-flex-fields", "gh_version": "\"1.0.2\",", "pypi_version": "1.0.2", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list", "error": "NA"}, {"project": "vincenty", "gh_version": "0.1.4,", "pypi_version": "0.1.4", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list setup.py", "error": "NA"}, {"project": "blaseball-mike", "gh_version": "6.11.0,", "pypi_version": "6.11.0", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list tests", "error": "NA"}, {"project": "eyecu-bumblebee", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No setup.py found"}, {"project": "flake8-annotations", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No setup.py found"}, {"project": "naturalsort", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "funcx-endpoint", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No setup.py found"}, {"project": "stable-baselines", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "nfl-insights", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: Could not clone the repo"}, {"project": "aio-geojson-geonetnz-quakes", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "sumologic-appclient-sdk", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: Could not clone the repo"}, {"project": "django-nested-inline", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "aprslib", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "async-google-trans-new", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "subarulink", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "rq", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: Could not clone the repo"}, {"project": "cget", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "parsedatetime", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "django-graphql-jwt", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No setup.py found"}, {"project": "pyloudnorm", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "lm-dataformat", "gh_version": "\"0.0.20\",", "pypi_version": "0.0.20", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list lm_dataformat test", "error": "NA"}, {"project": "zthreading", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "django-simple-history", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found in setup.py"}, {"project": "crashtest", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No setup.py found"}, {"project": "fcache", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No setup.py found"}, {"project": "pysonos", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "css-inline", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: Could not clone the repo"}, {"project": "tksheet", "gh_version": "6.2.1,", "pypi_version": "6.2.1", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list setup.cfg", "error": "NA"}, {"project": "django-username-email", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: Could not clone the repo"}, {"project": "mongo", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "pixivpy", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found in setup.py"}, {"project": "downloadutil", "gh_version": "1.0.0,", "pypi_version": "1.0.2", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list downloadutil setup.py", "error": "NA"}, {"project": "moclaphar", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No setup.py found"}, {"project": "bencoder-pyx", "gh_version": "3.0.1,", "pypi_version": "3.0.1", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list setup.cfg", "error": "NA"}, {"project": "emtwo-extensions", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found in setup.py"}, {"project": "flake8-html", "gh_version": "0.4.3,", "pypi_version": "0.4.3", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list setup.cfg", "error": "NA"}, {"project": "wikia", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "telethon-cosmosdb-sqlsession", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: Could not clone the repo"}, {"project": "parso", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "mdf-toolbox", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "text-fabric", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No setup.py found"}, {"project": "ipadic", "gh_version": "1.0.0,", "pypi_version": "1.0.0", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list README.md ipadic setup.py", "error": "NA"}, {"project": "pbiapi", "gh_version": "\"0.2.2\",", "pypi_version": "0.2.2", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list setup.py", "error": "NA"}, {"project": "test-helper", "gh_version": "0.2,", "pypi_version": "0.2", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list", "error": "NA"}, {"project": "yara-python", "gh_version": "4.3.0,", "pypi_version": "4.3.1", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list setup.cfg setup.py yara", "error": "NA"}, {"project": "toxiproxy-python", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "dictproxyhack", "gh_version": "1.1,", "pypi_version": "1.1", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list dictproxyhack.py setup.py", "error": "NA"}, {"project": "azure-cli-cdn", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No setup.py found"}, {"project": "bcolz", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found in setup.py"}, {"project": "edx-enterprise", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "dislevel", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: Could not clone the repo"}, {"project": "types-mypy-extensions", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No setup.py found"}, {"project": "soappy", "gh_version": "0.12.23.dev0,", "pypi_version": "0.12.5", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list .gitignore CHANGES.txt setup.py src tests README.txt ChangeLog", "error": "NA"}, {"project": "pytube", "gh_version": "__version__,  # noqa: F821", "pypi_version": "15.0.0", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list pytube tests", "error": "NA"}, {"project": "delphi-epidata", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No setup.py found"}, {"project": "pysegbase", "gh_version": "\"1.11.0\",", "pypi_version": "1.2.46", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list .travis.yml README.md examples requirements_conda.txt setup.cfg setup.py tests INSTALL", "error": "NA"}, {"project": "pcodedmp", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "invenio-mail", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found in setup.py"}, {"project": "django-file-resubmit", "gh_version": "0.5.2,", "pypi_version": "0.5.2", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list README.rst file_resubmit tests", "error": "NA"}, {"project": "mkdocs-techdocs-core", "gh_version": "\"1.2.1\",", "pypi_version": "1.2.1", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list", "error": "NA"}, {"project": "rindcalc", "gh_version": "v3.0.0,", "pypi_version": "3.0.0", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list README.md rindcalc tests", "error": "NA"}, {"project": "haralyzer", "gh_version": "2.3.0,", "pypi_version": "2.3.0", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list setup.cfg", "error": "NA"}, {"project": "jenkins-view-builder", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found in setup.py"}, {"project": "pines", "gh_version": true, "pypi_version": true, "status": "Failed", "diffoscope:": "NA", "error": "Not Error from GithHub PyPi: Could not fetch PyPi Source"}, {"project": "grs", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "pyecog", "gh_version": "0.2.3,", "pypi_version": "0.2.3", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list pyecog", "error": "NA"}, {"project": "aws-cdk-aws-appflow", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No setup.py found"}, {"project": "azure-storage-file", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No setup.py found"}, {"project": "dcoscli", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "coinor-dippy", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No setup.py found"}, {"project": "dask-xgboost", "gh_version": "\"0.2.0\",", "pypi_version": "0.2.0", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list setup.cfg", "error": "NA"}, {"project": "base45", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No setup.py found"}, {"project": "django-concurrency", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "codecheck", "gh_version": "1.3.1,", "pypi_version": "1.3.1", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list codecheck", "error": "NA"}, {"project": "pymilvus-test", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found in setup.py"}, {"project": "py-triton", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "mypy-boto3-customer-profiles", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No setup.py found"}, {"project": "cexprtk", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "aws-cdk-aws-databrew", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: Could not clone the repo"}, {"project": "telebot", "gh_version": "0.0.6,", "pypi_version": "0.0.5", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list setup.cfg setup.py", "error": "NA"}, {"project": "srv-hijacker", "gh_version": "\"0.0.9\",", "pypi_version": "0.0.9", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list", "error": "NA"}, {"project": "cryptosteganography", "gh_version": "0.8.2,", "pypi_version": "0.8.3", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list README.rst requirements.txt setup.cfg setup.py src", "error": "NA"}, {"project": "django-extended-choices", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found in setup.py"}, {"project": "tencentcloud-sdk-python-tbm", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "ovoenergy", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found in setup.py"}, {"project": "emora-stdm", "gh_version": "2.0.5,", "pypi_version": "2.0.5", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list emora_stdm", "error": "NA"}, {"project": "kruxstatsd", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "mwoauth", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "bc-python-hcl2", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No setup.py found"}, {"project": "pydeconz", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "aws-solutions-constructs-aws-events-rule-sqs", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No setup.py found"}, {"project": "testfixtures", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "json-tricks", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found in setup.py"}, {"project": "adafruit-pureio", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No setup.py found"}, {"project": "gittyleaks", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "django-click", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "eth-typing", "gh_version": "\"3.4.0\",", "pypi_version": "3.4.0", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list MANIFEST.in", "error": "NA"}, {"project": "claptcha", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: Could not clone the repo"}, {"project": "nylas", "gh_version": "Failed", "pypi_version": "Failed", "status": "Failed", "diffoscope:": "NA", "error": "GH: No version found after checking __init__.py"}, {"project": "subword-nmt", "gh_version": "0.3.8,", "pypi_version": "0.3.8", "status": "Success", "diffoscope:": "\u251c\u2500\u2500 file list README.md subword_nmt", "error": "NA"}]}